__tests__/services/TabEvents.spec.ts

```diff
@@ -28,19 +28,21 @@ import StoreUser from '../../src/services/StoreUser';
 jest.requireActual('../../src/services/AlarmEvents');
 
 // This dynamically generates the spies for all functions in AlarmEvents
-const spyAlarmEvents: {[s: string]: jest.SpyInstance} = {};
+const spyAlarmEvents: { [s: string]: jest.SpyInstance } = {};
 for (const k of Object.keys(AlarmEvents)) {
```

Comment:
As said previously, last PR moved the spy generator into setup.js.  With the formatting difference, you'll need to adjust it yourself.